wiki/Beatmap_Editor/AiMod/de.md

```diff
@@ -0,0 +1,175 @@
+AiMod
+========
+
+[AiMod](AiMod "wikilink") ist ein Programm, welchen man im Beatmap Editor finden kann.
+Der [AiMod](AiMod "wikilink") kann über den Tastenkürzel **\[Strg\] + \[Shift\] + \[A\]** oder über **File**, dann **Open AiMod** geöffnet werden.
```

Comment:
same thing (remove link; change `[AiMod](AiMod "wikilink")` to `AiMod`)

---

for keyboard keys, do not surround the key with square brackets. Instead, use code (this: `` ` ``).
So it should be:
`Strg` + `Shift` + `A`
`` `Strg` + `Shift` + `A` ``